blocks/library/latest-posts/index.js

```diff
@@ -50,23 +91,62 @@ registerBlockType( 'core/latestposts', {
 				);
 			}
 
-			return (
-				<div className="blocks-latest-posts">
-					<ul>
+			const { focus, instanceId } = this.props;
+			const { displayPostDate } = this.props.attributes;
+
+			const displayPostDateId = `post-date-toggle-${ instanceId }`;
+			const postToShowId = `post-to-show-${ instanceId }`;
+
+			return [
+				focus && (
+					<InspectorControls key="inspector">
+						<div className="editor-latest-posts__row">
+							<label htmlFor={ displayPostDateId }>{ __( 'Display post date' ) }</label>
+							<FormToggle
+								id={ displayPostDateId }
+								checked={ displayPostDate }
+								onChange={ this.toggleDisplayPostDate }
+								showHint={ false }
+							/>
+						</div>
+						<div className="editor-latest-posts__row">
+							<label htmlFor={ postToShowId }>{ __( 'Number of posts to show' ) } </label>
+							<input
+								type="text"
+								value={ this.props.attributes.poststoshow }
+								ref={ postToShowId }
+								id={ postToShowId }
+								className="editor-latest-posts__input"
+								onChange={ () => this.changePostsToShow( this.refs[ postToShowId ].value ) }
+							/>
+						</div>
```

Comment:
We should absolutely use the same controls everywhere, but whether this happens post merge or not I don't think is important. Should I open a separate ticket for making sure controls are all the same?